src/fontra/views/editor/editor.js

```diff
@@ -2119,10 +2119,11 @@ export class EditorController {
 
     dialog.setContent(contentElement);
 
-    setTimeout(
-      () => contentElement.querySelector("#anchor-name-text-input")?.focus(),
-      0
-    );
+    setTimeout(() => {
+      const inputName = contentElement.querySelector("#anchor-name-text-input");
```

Comment:
This is a poor variable name. This is not the "inputName" (suggesting a string), but a DOM element. Please change to `inputNameElement`.

Reply:
Yes, you're right. Sorry. I change it.